### PR TITLE
Adds hardcoded version and adds github workflow files

### PR DESCRIPTION
**Pull request by saltiyazan**
- Adds the `version` keyword to the snapcraft.yaml file and uses a hardcoded version.
- Updates the repository to use the shared Github Actions. (Snapcrafters/ci)

### CODE REVIEW

**Comment by saltiyazan** [comment]
Hi @merlijn-sebrechts 
Here is my first PR to Snapcrafters, I followed the contributing guide to add the workflows.
Could you please take a look?

File: .github/workflows/release-to-candidate.yml
Comment (jnsgruk) [inline .github/workflows/release-to-candidate.yml]: I don't think this is likely to be useful in a CLI application - I think it can be removed :)

**Review by jnsgruk — DISMISSED** [review]
LGTM - probably worth removing the `get-screenshot` action before we merge. Once you've got that sorted.

I'll add the credentials to the repo now.

**Comment by jnsgruk** [comment]
Could you please also add a README using the template shown in https://github.com/snapcrafters/discord?

File: snap/snapcraft.yaml
Comment (jnsgruk) [inline snap/snapcraft.yaml:14]:
```suggestion
architectures:
  - amd64
  - arm64
  - armhf
  - ppc64el
  - s390x
```

The snap currently has releases across all of these architectures :) I don't think anyone is actually using the `ppc64el` or `s390x` releases (according to the metrics), but we should probably leave `armhf` until it causes us problems.

File: .github/workflows/release-to-candidate.yml
Comment (saltiyazan) [reply .github/workflows/release-to-candidate.yml]: ✅

File: snap/snapcraft.yaml
Comment (saltiyazan) [reply snap/snapcraft.yaml:14]: ✅

**Comment by saltiyazan** [comment]
> Could you please also add a README using the template shown in https://github.com/snapcrafters/discord?

I added it, I'm not fully sure about the maintainers and license sections (I added the LICENSE file too) :)

File: README.md
Comment (jnsgruk) [inline README.md]:
```suggestion
<a href="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yml"><img src="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yml/badge.svg"></a>
```

**Review by jnsgruk — DISMISSED** [review]

**Review by jnsgruk — APPROVED** [review]

**Comment by jnsgruk** [comment]
Thanks @saltiyazan!

**Pull request merged**
